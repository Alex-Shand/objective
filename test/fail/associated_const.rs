#[objective::objective]
trait Test {
    const CONST: usize;
}

fn main() {}
