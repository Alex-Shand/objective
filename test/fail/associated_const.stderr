error: Objective only supports types and functions in traits
 --> test/fail/associated_const.rs:3:5
  |
3 |     const CONST: usize;
  |     ^^^^^
